libevmasm/Instruction.cpp

```diff
@@ -326,6 +333,9 @@ static std::map<Instruction, InstructionInfo> const c_instructionInfo =
 	{Instruction::LOG2,           {"LOG2",            0,  4,   0,  true,       Tier::Special}},
 	{Instruction::LOG3,           {"LOG3",            0,  5,   0,  true,       Tier::Special}},
 	{Instruction::LOG4,           {"LOG4",            0,  6,   0,  true,       Tier::Special}},
+	{Instruction::RETF,           {"RETF",            0,  0,   0,  true,       Tier::Special}},
+	{Instruction::CALLF,          {"CALLF",           2,  0,   0,  true,       Tier::Special}},
+	{Instruction::JUMPF,          {"JUMPF",           2,  0,   0,  true,       Tier::Special}},
```

Comment:
These should get dedicated tiers too, [to match execution specs](https://github.com/ethereum/execution-specs/blob/eips/prague/eof/src/ethereum/prague/vm/gas.py#L72-L74).

Reply:
So we should have it also for DATALOADN. https://github.com/ethereum/execution-specs/blob/eips/prague/eof/src/ethereum/prague/vm/gas.py#L78

Reply:
True. You could change that here or submit a separate PR to change that.

Reply:
BTW, one more thing that we should do for `DATALOADN` is `hasInstruction()`. And then it should be going into the branches that handle `Operation` in those `AssemblyItem` helpers that return various properties like number of returns.